dnf/cli/commands/repolist.py

```diff
@@ -116,7 +116,16 @@ def run(self):
         on_hiend = term.MODE['normal']
         tot_num = 0
         cols = []
-        if not repos:
+
+        if repos:
+            if arg not in ('disabled', 'all') and not enabled_repos:
+                logger.warn(_('No repositories enabled'))
+                return
+            elif arg == 'disabled' and \
+                    not any(drepo.enabled is False for drepo in repos):
+                logger.warn(_('No repositories disabled'))
```

Comment:
The question is if we should report it into logger. The output is also only printed to terminal but in this case it should be reported into stderror.